app.js

```diff
@@ -9,10 +9,13 @@ const session = require( 'express-session' )
 const models = require( './models/index' )
 const passport = require( './auth/passport' )
 const protectRoute = require( './auth/protectRoute' )
+const checkToken = require( './auth/checkToken' )
 
 const routes = require( './routes/index' )
 const accounts = require( './routes/accounts' )
 const items = require( './routes/items' )
+const apiAccounts = require( './routes/api/accounts')
```

Comment:
I would like to change this to a manifest file so we can separate by version without having to import a ton of files:
```js
const api = require( './routes/api/manifest' ).v1
// Later:
app.use( '/api/v1', api )
```
The manifest file will just require the necessary files and then re-export them, but I'm not sure what this looks like for routes, so some research will be necessary.  Something like:
```js
const accounts = require( './v1/accounts' )

module.exports = {
  v1: { accounts, /* ... others ... */ },
  v2: { /* ... when we have v2 endpoints ... */ }
}
```
Not sure how scalable this will be, but it helps with organization for now.